_wiki/FAQ/Windows.md

```diff
@@ -5,186 +5,157 @@ hide_from_auto_list: true
-program, although there may be better ways to do it for those programs.
+- Right click `OpenTabletDriver.UX.Wpf.exe` and select `Create shortcut`.
+- Press <kbd>Win</kbd> + <kbd>R</kbd> to open the Run dialog.
+- Type `shell:startup` then press <kbd>Enter</kbd>.
+- Move the shortcut you created earlier to the folder that opened.
 
- - Right click `OpenTabletDriver.UX.Wpf.exe` and create a shortcut.
- - Press <kbd>Win</kbd> + <kbd>R</kbd> and type in `shell:startup`
- - Move the shortcut into the folder that opened.
+To start OpenTabletDriver minimised, continue below:
 
-Starting OpenTabletDriver minimised can also be done by changing the properties of a shortcut.
-To do this, right click the OpenTabletDriver shortcut, go to properties > run > minimised.
+- Right click the shortcut and select `Properties`.
+- In the `Run` dropdown, select `Minimized`.
 
 ---
 
-### Plugin Manager won't open/I get a warning on startup? {#administrator}
+### I get a warning about not running it as Administrator on startup? {#administrator}
 
-When Starting OpenTabletDriver you may get an error warning you about Administrator Permissions.
-OpenTabletDriver should NEVER be run as Administrator or with Administrator permissions.
-This causes the plugin manager, FAQ page, and Tablet debugger to not work, and can interfere with several other features.
+OpenTabletDriver should *NEVER* be run as Administrator, and there is never a need to run as Administrator.
 
-The following instructions should fix your issue under normal circumstances, but if you're using the (disabled in a fresh Windows install by default) account called "Administrator",
-usually because you're using a "custom" iso (unofficial version of Windows, also sometimes referred to as a "debloated Windows") install,
-these instructions will not work for you.
+Follow the [instructions](https://learn.microsoft.com/en-us/mem/intune/user-help/you-need-to-enable-uac-windows#adjust-protection-level) provided by Windows and adjust protection level to at least:
 
-We generally recommend against using altered or 3rd party versions of Windows,
-as OTD is likely to not be the only software adversely affected by the changes these custom ISOs make.
+- Notify me when apps try to make changes to my computer.
 
- - Press the Windows key and search` User Account Control`.
- - Move the slider to one of the top 2 options.
- - Click OK, then restart your computer.
+Then restart your computer to apply.
 
----
+If you are still getting the warning, make sure that you are not using the account called `Administrator`. This is *NEVER* meant to be used as a normal account for daily use. Create a new normal account with administrator privileges and use that instead.
 
-### Why is my interface completely blank? {#rtss}
+> Note: We are aware that most custom ISOs use the Administrator account. We *do not* provide support to those that use custom ISOs.
```

Comment:
should probably explain this a bit better than what reads as "fuck you" (to me, anyway)

maybe say something about the system being unpredictable and we don't know what has changed that we might otherwise take for granted